string moseg1
string moseg2
